gallery/src/styles/NxPageTitle/NxPageTitleEverythingExample.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTag, NxDropdown, useToggle } from '@sonatype/react-shared-components';
+
+function NxPageTitleEverythingExample() {
+  const [isOpen, onToggleCollapse] = useToggle(false);
+
+  return (
+    <div className="nx-page-title">
+      <hgroup className="nx-page-title__headings">
+        <h1 className="nx-h1">
+          Longer Page Title Foo Bar Foo!
+        </h1>
+        <h2 className="nx-h2 nx-page-title__sub-title">
+          This is a page sub-title that will truncate with an ellipsis
+        </h2>
+      </hgroup>
+      <div className="nx-btn-bar">
+        <NxDropdown label="Example Dropdown" isOpen={isOpen} onToggleCollapse={onToggleCollapse}>
+          <a href="#" className="nx-dropdown-button">
+            <span className="nx-dropdown-button-content">Text Link 1</span>
+          </a>
+          <a href="#" className="nx-dropdown-button">
+            <span className="nx-dropdown-button-content">Text Link 2</span>
+          </a>
+          <a href="#" className="nx-dropdown-button">
+            <span className="nx-dropdown-button-content">Text Link 3</span>
+          </a>
+          <a href="#" className="nx-dropdown-button">
+            <span className="nx-dropdown-button-content">Text Link 4</span>
+          </a>
+        </NxDropdown>
+      </div>
+      <div className="nx-page-title__description">
+        <p className="nx-p">This is a page description.</p>
+        <p className="nx-p">
+          jeans sign papier-mache assassin San Francisco rifle physical 3D-printed denim tanto courier concrete dolphin
+          rebar free-market. tank-traps papier-mache dead free-market tanto drone concrete dolphin sunglasses weathered
+          dead jeans office vehicle nodal point. motion film meta- monofilament knife vinyl post- bridge jeans city
+          Tokyo alcohol marketing girl vehicle.
+        </p>
+      </div>
+      <div className="nx-page-title__tags">
```

Comment:
This would be a good spot to demonstrate a mix of `NxTag`s and `NxPolicyViolationIndicator`s

Reply:
> This would be a good spot to demonstrate a mix of `NxTag`s and `NxPolicyViolationIndicator`s

I debated that but I didn't think that we'd ever mix them in one line/container, and I'm not of the opinion that we should demonstrate that. IMO if a page title has both tags and indicators they should be in separate containers / on separate lines.

Reply:
> I debated that but I didn't think that we'd ever mix them in one line/container, and I'm not of the opinion that we should demonstrate that. IMO if a page title has both tags and indicators they should be in separate containers / on separate lines.

The mockup shows it:
![image](https://user-images.githubusercontent.com/3630091/116089347-c06f4e00-a670-11eb-8dc9-46251a52ed27.png)

Reply:
@dturner32 is this a design direction that we can/will mix tags, indicators, etc on a line?

Reply:
> @dturner32 is this a design direction that we can/will mix tags, indicators, etc on a line?

yes, that is the intention

Reply:
> > @dturner32 is this a design direction that we can/will mix tags, indicators, etc on a line?
> 
> yes, that is the intention

Mixed example: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/2f4ef4d8b695bc63eb54ad3908347a7878512aa4

Reply:
> Mixed example: 2f4ef4d

The mockups specify 24px between a violation indicator and tags.  Which begs some questions - does that apply only at the switchover from one type to the other? What about in the opposite direction, going from tags to violation indicators?  Or is that ordering just unsupported?

Reply:
@derrek?

Also can other indicators appear here? `NxThreatCounter` for example? Or the mysterious "Distributed" label that's still in the mockups? 

What happens if `NxTag`s are mixed with `NxPolicyViolationIndicator`s?

Reply:
> > Mixed example: [2f4ef4d](https://github.com/sonatype/sonatype-react-shared-components/commit/2f4ef4d8b695bc63eb54ad3908347a7878512aa4)
> 
> The mockups specify 24px between a violation indicator and a tag. Which begs some questions - does that apply only at the switchover from one type to the other? What about in the opposite direction, going from tags to violation indicators? Or is that ordering just unsupported?

the 24px remains the standard spacing between the indicator groupings and tag groupings and only applies when going from indicators to tags, otherwise the spacing between tags remains the same (8px) and the spacing between indicators remains the same (16px). Indicators will always show up first in terms of order. If there are threat counts, those will show up second, and then tags third.

Reply:
> @derrek?
> 
> Also can other indicators appear here? `NxThreatCounter` for example? Or the mysterious "Distributed" label that's still in the mockups?
> 
> What happens if `NxTag`s are mixed with `NxPolicyViolationIndicator`s?

Other indicators can also appear (i.e. threat counts). The order would be: policy indicators, threat counts, tags. Indicators and tags will remain grouped and never be mixed together, in other words, tags will never show up in the same grouping as indicators (i.e. indicator, tag, indicator)

Reply:
> > @derrek?
> > Also can other indicators appear here? `NxThreatCounter` for example? Or the mysterious "Distributed" label that's still in the mockups?
> > What happens if `NxTag`s are mixed with `NxPolicyViolationIndicator`s?
> 
> Other indicators can also appear (i.e. threat counts). The order would be: policy indicators, threat counts, tags. Indicators and tags will remain grouped and never be mixed together, in other words, tags will never show up in the same grouping as indicators (i.e. indicator, tag, indicator)

https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/5f080cb9f8ac77ddbb116d5e0bfa32745b4994d7

I resisted creating wrapping containers for `NxPolicyViolationIndicator`s and `Nxtag`s (`NxThreatCounter`s are wrapped in a `dl`), to keep the complexity down.